src/styles.scss

```diff
@@ -1,8 +1,14 @@
 /* You can add global styles to this file, and also import other style files */
 @import '~@angular/material/prebuilt-themes/indigo-pink.css';
 
-html,
 body {
-  margin: 0;
-  font-family: Roboto, Arial, sans-serif;
+  background: rgba(0, 0, 0, 0.03);
+
+  &,
+  html {
```

Comment:
Why is the html tag below the body tag?

Reply:
It's actually on the same level (it's separated by a comma and not only by white spaces). The nested write style was chosen to avoid duplication.